### ISSUE TITLE
Keyup / keypress event is not firing in Edge (autolink plugin)

### ISSUE THREAD

**Issue by freeads777** [tinymce/tinymce#5076]
I am trying to figure this one out in tinymce (4.7). When you type a link and then press the space bar all browsers will convert that link into a URL.

However, in Edge you have to use Enter to do the same. The space bar won't do that. Is there an Edge browser specific command for the space bar? I'd like to have uniformity for all browsers. Basically in Edge the autolink plugin doesn't convert links into clickable URL unless you press Enter RIGHT AFTER you typed the link. If you don't do that the link will remain as text.

I am thinking this is the part of the code responsible for that:
```
...
var o, a = /^(https?:\/\/|ssh:\/\/|ftp:\/\/|file:\/|www\.|(?:mailto:)?[A-Z0-9._%+\-]+@)(.+)$/i;
return e.settings.autolink_pattern && (a = e.settings.autolink_pattern), e.on("keydown", function(t) {
    if (13 == t.keyCode) return r(e)
}), tinymce.Env.ie ? void e.on("focus", function() {
    if (!o) {
        o = !0;
        try {
            e.execCommand("AutoUrlDetect", !1, !0)
        } catch (e) {}
    }
}) : (e.on("keypress", function(n) {
    if (41 == n.keyCode) return t(e)
}), void e.on("keyup", function(t) {
    if (32 == t.keyCode) return n(e)
}))
...
```

**Comment by ltrouton** [comment]
Hi @freeads777. Thanks for the report! I can confirm that this is a bug and have a fix in the works. We will let you know when we have an update.

**Comment by lnewson** [comment]
Sorry for missing this, but this issue was fixed in TinyMCE 5.0.12, which was released almost a month ago now. Please let us know if you have any further issues.
